src/network/serveropcodes.cpp

```diff
@@ -155,71 +152,95 @@ const ClientCommandFactory clientCommandFactoryTable[TOCLIENT_NUM_MSG_TYPES] =
 	null_command_factory, // 0x1D
 	null_command_factory, // 0x1E
 	null_command_factory, // 0x1F
-	{ "TOCLIENT_BLOCKDATA",                2, true }, // 0x20
-	{ "TOCLIENT_ADDNODE",                  0, true }, // 0x21
-	{ "TOCLIENT_REMOVENODE",               0, true }, // 0x22
+	{ "TOCLIENT_BLOCKDATA",                MTSCMC_MAP, true }, // 0x20
```

Comment:
So these three were using 2 channels before, I think we should keep them on separate channels, if only to indicate that there is no ordering assumption between theses.

Reply:
Perhaps another channel MT_SCMC_MAPCHANGE.

Reply:
Makes sense. And hey, it's pretty easy to change this stuff now.